### PR TITLE
python extraction script for data extraction 

### PR DESCRIPTION
**Pull request by sujanparab12**
python extraction script for data extraction and uploading directly to s3

### CODE REVIEW

File: scripts/python/extract_data.py [hunk 1]
```diff
@@ -1,97 +1,107 @@
...
-
-
-
-
-
-
-
-
+    # S3 Bucket details 
+    bucket_name = 'eltproject'
```
Comment (omingale0707) [inline scripts/python/extract_data.py:87]: Move this to config file

File: scripts/python/extract_data.py [hunk 2]
```diff
@@ -1,97 +1,107 @@
...
-if __name__ == "__main__":
-   
-    config = read_config(file_path)               # Read parameters from the configuration file
-
-    info = read_info(file_path_2)
-
+    def load_credentials(self):
+        # Extract AWS credentials from the loaded credentials
+        aws_access_key_id = self.config['keys']['access_key_id']
+        aws_secret_access_key = self.config['keys']['secret_access_key']
```
Comment (omingale0707) [inline scripts/python/extract_data.py:48]: make sure the credential attributes are hidden

File: scripts/python/extract_data.py [hunk 3]
```diff
@@ -1,97 +1,107 @@
...
-def get_sql_connection(config):
-    username = config['source']['username']
-    password = config['source']['password']
-    server = config['source']['server']
-    database = config['source']['database']
+    def get_sql_connection(self):
+        username = self.config['source']['username']
+        password = self.config['source']['password']
+        server = self.config['source']['server']
+        database = self.config['source']['database']
```
Comment (omingale0707) [inline scripts/python/extract_data.py:22]: make sure the credential attributes are hidden

File: scripts/python/extract_data.py [hunk 4]
```diff
@@ -1,97 +1,107 @@
...
-file_path_2 = 'C:\\elt\\info.yaml'
 
-def read_config(file_path):
-    with open(file_path, 'r') as file:
-        config = yaml.safe_load(file)
-    return config
+class SQLServerData:
+    def __init__(self, credentials, config_path):
+        self.config1 = self.read_config(config_path)
+        self.config = self.read_config(credentials)
```
Comment (omingale0707) [inline scripts/python/extract_data.py:11]: name the variables in a better way

**Comment by omingale0707** [comment]
Add the changes you made to the config and key files, so that the updated format is available

**Review by omingale0707 — APPROVED** [review]

**Review by OmkarShevde — APPROVED** [review]

**Pull request merged**